components/financial_assistance/app/controllers/financial_assistance/applicants_controller.rb

```diff
@@ -131,6 +131,11 @@ def immigration_document_options
       # @country = vlp_docs.detect{|doc| doc.subject == vlp_doc_subject }.try(:country_of_citizenship) if vlp_docs
     end
 
+    def destroy
+      flash[:error] = build_error_messages(@model) unless @applicant.update_attributes(is_active: false)
```

Comment:
At this point in EA when we delete(deactivate) a Family member, what happens in the present system(assumption is applicant object gets delete)?
 -> If so we need to have the consistency by deleting the applicant object instead of deactivating it.

Reply:
Deleting applicant object.